google-cloud-bigtable/pom.xml

```diff
@@ -351,6 +352,8 @@
             <configuration>
               <forkCount>1</forkCount>
               <parallel>none</parallel>
+              <redirectTestOutputToFile>true</redirectTestOutputToFile>
+              <reportsDirectory>${project.build.directory}/test-reports</reportsDirectory>
```

Comment:
this need to be unique per integration test profile so that the profiles dont clobber each other